src/simulator/views/map/map_view.py

```diff
@@ -311,13 +320,7 @@ def HDScreenShot(self):
 
     def cube_center(self, p: Point) -> Point:
         x, y, z = self.to_point3(p)
-        x += 0.5
-        y += 0.5
-        if self._services.algorithm.map.size.n_dim == 3:
-            z -= 0.5
-        else:
-            z = 0.1 # have overlay be slightly above surface for 2D maps
-        return Point(x, y, z)
+        return Point(x + 0.5, y + 0.5, z - 0.5)
```

Comment:
what r these magic numbers in the first place

Reply:
Finding the centre of a cube, like the name of the function suggests

Reply:
oh lmfao nvm